### PR TITLE
Shares

### PR DESCRIPTION
**Pull request by Sensationull**
Added shares functionality so that it can actually work

### CODE REVIEW

**Review by htlin2 — COMMENTED** [review]
Highly recommend adding bundle.js in the .gitignore

I will do that immediately

File: client/src/components/Form.jsx [hunk 1]
```diff
@@ -31,8 +46,8 @@ class Form extends React.Component {
     return <div>
       <h3 id="company">{this.state.company}</h3>
       <MarketPrice prices={this.state.marketPrice}/>
-      <input placeholder="Shares"></input>
-      <EstimatedCost estimation={this.state.estimatedCost}/>
+      <input type="number" placeholder="Shares" onChange={this.handleShares}></input>
```
Comment (htlin2) [inline client/src/components/Form.jsx:49]: input can be self closing tags for cleaner look. 
Example:
< input / >

Comment (htlin2) [reply client/src/components/Form.jsx]: // <input type="number" placeholder="Shares" onChange={this.handleShares}>

File: client/src/index.js [hunk 2]
```diff
@@ -16,7 +16,6 @@ class App extends React.Component {
   asyncGet(callback) {
     axios.get('/api/buytest')
       .then(function (response) {
```
Comment (htlin2) [inline client/src/index.js:18]: Maybe change function (response) to ES6 arrow functions for consistency look?

File: package.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (htlin2) [inline package.json:8]: test script can be removed if it is not being used.

Comment (Sensationull) [reply package.json:8]: I wrote about this in my Journal haha I knew I needed to get rid of one of them. I'll do that in the next PR

**Pull request merged**